src/main/java/com/aviva/CarRecommendations/InterestFilter.java

```diff
@@ -0,0 +1,65 @@
+package com.aviva.CarRecommendations;
+
+import com.aviva.DataAccess.BankingDataProcess;
+import com.aviva.Entities.AccountHolder;
+import com.aviva.Entities.Car;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class InterestFilter {
+
+    public JSONObject getBestFiveCars(String accountNumber) {
```

Comment:
this method is pretty long, we should probably split it up a bit. I've included some suggestions

Reply:
we can't realistically shorten it aside from few efficiency changes which we dont have time for right now